docs/en/reference/dql-doctrine-query-language.rst

```diff
@@ -1392,7 +1392,7 @@ Result Cache API:
     $result = $query->getResult(); // saved in given result cache id.
 
     // or call useResultCache() with all parameters:
-    $query->useResultCache(true, $seconds = 3600, 'my_query_result');
+    $query->useResultCache(true, seconds: 3600, 'my_query_result');
```

Comment:
This would be lifetime too.

```suggestion
    $query->useResultCache(true, lifetime: 3600, 'my_query_result');
```

Reply:
This method is deprecated since 2.7

should the documentation still promote its use?

Reply:
I've asked as I'd like to remove this reference, or should we fix it now and remove it in the future?

Reply:
Yes, we should document the non-deprecated method here.

Reply:
I've asked because method `userResultCache` is deprecated, and this section is written in a language that kind of 'promotes' it's use. I'd suggest to replace these lines with  suggested replacement `enableResultCache`, but that's for another PR

https://github.com/doctrine/orm/blob/9f555ea8fb4f7ae9a362285dc71e85776a1c5cc3/lib/Doctrine/ORM/AbstractQuery.php#L709-L726

Reply:
Yes, please send a PR. 🙂